packages/toolpad-app/src/utils/ga.ts

```diff
@@ -17,3 +18,12 @@ export const reportWebVitalsToGA = ({ id, label, name, value }: NextWebVitalsMet
     });
   }
 };
+
+export const sendAppCreatedEvent = (name: string, templateId?: AppTemplateId): void => {
+  if (config.gaId) {
+    window.gtag('event', 'app_created', {
```

Comment:
Would it make sense to namespace those `toolpad_app_created`?

Reply:
Why not, would be a good standard for custom events, will add now.